contracts/0.4.24/oracle/LidoOracle.sol

```diff
@@ -328,6 +338,25 @@ contract LidoOracle is ILidoOracle, AragonApp {
         return (minReportableEpochId, getCurrentEpochId());
     }
 
+    /**
+     * @notice Reports beacon balance and its change
+     */
+    function getLastCompletedReports()
+        public view
+        returns (
+            uint256 lastTotalPooledEther,
+            uint256 prevTotalPooledEther,
+            uint256 timeElapsed
+        )
+    {
+        BeaconSpec memory beaconSpec = _getBeaconSpec();
+        CompletedReport storage last = lastCompletedReports[COMPLETED_REPORT_ID_LAST];
+        CompletedReport storage prev = lastCompletedReports[COMPLETED_REPORT_ID_PREV];
+        lastTotalPooledEther = last.totalPooledEther;
+        prevTotalPooledEther = prev.totalPooledEther;
```

Comment:
is the difference between `lastTotalPooledEther` and `prevTotalPooledEther` profit?

Reply:
I think yes, as we discussed. From the steth holder point of view.

Reply:
As discussed, updated the code to avoid accounting sold stETH during the epoch. TODO: try to avoid 2 calls to `Lido.sol` per frame.